### PR TITLE
Update links in automatic issues

### PR DESCRIPTION
**Pull request by KatherineCox**
I'm starting a new book, and realized these still link to the wiki.  I'll update the links to point to ottrproject.org as I work through the steps.

- Updates links in the automatic issue `set-repo-settings`
- Instructions to set up GH_PAT were listed twice, so I removed the less detailed one.
- Updates links in automatic issue `update-enrollment`
- Updates (most) links in automatic issue `templates-to-edit`

### CODE REVIEW

**Comment by KatherineCox** [comment]
@cansavvy
- I wasn't able to find a corresponding page on ottrproject.org for quite all of the wiki page links, but this at least gets most of them
- looks like there are instructions on setting up GH_PAT in both `git-secrets.md` and `set-repo-settings.md`.  I updated the link in `set-repo-setting` since that's where I saw it first.  Not sure if you want to delete `git-secrets.md`, or keep it and remove the GH_PAT instructions from `set-repo-settings`.

File: .github/automatic-issues/set-repo-settings.md [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
   `Name`:  `GH_PAT`
   `value`: A personal access token [following these instructions](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-token).
   Underneath `Select scopes`, check both `repo` and `workflow`.
   Then copy the PAT and save as the value.
+  
+- [ ] GitHub pages is turned on
+  - [ ] Go to `Settings` > `Pages`. Underneath `Source`, choose `main` for the branch and select the `docs` folder. Then click `Save`.  
+  - [ ] Check `Enforce HTTPS`.
 
 - [ ] [Set branch protections settings](https://github.com/jhudsl/OTTR_Template/wiki/Start-a-new-course#9-set-up-branch-rules)
```
Comment (cansavvy) [inline .github/automatic-issues/set-repo-settings.md]:
```suggestion
- [ ] [Set branch protections settings](https://www.ottrproject.org/getting_started.html#8_Set_up_branch_rules)
```

File: .github/automatic-issues/templates-to-edit.md [hunk 2]
```diff
@@ -12,23 +12,23 @@ The following files need to be edited to get this new course started!
...
 By default this course template will use the jhudsl data science lab style. However, you can customize and switch this to another style set.
 
 #### Using a style set
 
-[Read more about the style sets here](https://github.com/jhudsl/OTTR_Template/wiki/Change-Title-&-Customize-style#using-a-style-set).
+[Read more about the style sets here](https://www.ottrproject.org/customize-style.html#Using_a_style_set).
 
 - [ ] On a new branch, copy the `style-sets/<set-name>/index.Rmd` and `style-sets/<set-name>/_output.yml` to the top of the repository to overwrite the default `index.Rmd` and `_output.yml`.
 - [ ] Copy over all the files in the `style-sets/<set-name>/copy-to-assets` to the `assets` folder in the top of the repository.
 - [ ] [Create a pull request](https://github.com/jhudsl/OTTR_Template/wiki/Start-editing-your-course#getting-started-with-the-github-workflow) with these changes, and double check the rendered preview to make sure that the style is what you are looking for.
```
Comment (cansavvy) [inline .github/automatic-issues/templates-to-edit.md]:
```suggestion
- [ ] [Create a pull request](https://www.ottrproject.org/writing_content.html#Open_a_pull_request) with these changes, and double check the rendered preview to make sure that the style is what you are looking for.
```

**Comment by cansavvy** [comment]
Thanks for getting this started, @KatherineCox These updates were long overdue! I added another commit of link replacements. I think everything should be covered now. 🤞

**Comment by cansavvy** [comment]
@KatherineCox Can you give the changes I just added a review? The ones you did were good, so if we both think each other's changes are good then I say we merge this.

**Comment by KatherineCox** [comment]
@cansavvy Caught one more link and updated it.  Looks good to me!

**Comment by cansavvy** [comment]
Thanks, @KatherineCox ! I'm going to merge this then!

**Pull request merged**